pypeit/spectrographs/keck_kcwi.py

```diff
@@ -180,7 +182,12 @@ def default_pypeit_par(self):
         """
         par = pypeitpar.PypeItPar()
         par['rdx']['spectrograph'] = 'keck_kcwi'
-        # Set wave tilts order
+
+        # Subtract the detector pattern from all frames
+        #par['baseprocess']['use_pattern'] = True
```

Comment:
remove?

Reply:
I was going to ping you on this @profxj - I can't figure out why, but `baseprocess` can't be used to update this argument for all frames, so I have to put this into all frames manually. What is the preferred approach, using `baseprocess` or explicitly list each frametype?

Reply:
Does the baseprocess documentation here help? https://pypeit.readthedocs.io/en/latest/pypeit_par.html 

I've used this for e.g. use_illumflat and other things, so I'm not sure why it would fail for use_pattern.

Reply:
I think baseprocess onlyh works with pypeit files, sorry. @kbwestfall is the expert. In code, I think I've been setting it by hand using that function that X created.